custom_components/garmin_connect/__init__.py

```
@@ -431,14 +441,13 @@ async def _async_update_data(self) -> dict:
-                    "Too many requests error occurred during update: %s", err.response.text)
+                    "Too many requests error occurred during update: %s", err.response.text
+                )
                 return {}
-            _LOGGER.error(
-                "Unknown HTTP error occurred during update: %s", err)
+            _LOGGER.error("Unknown HTTP error occurred during update: %s", err)
             return False
         except Exception as err:  # pylint: disable=broad-except
-            _LOGGER.exception(
-                "Unknown error occurred during update: %s", err)
+            _LOGGER.exception("Unknown error occurred during update: %s", err)
```

Ruff (0.14.3)

Redundant exception object included in `logging.exception` call

(TRY401)